lib/common/acl.c

```diff
@@ -85,27 +85,20 @@ create_acl(xmlNode *xml, GList *acls, enum xml_private_flags mode)
     } else {
         GString *buf = g_string_sized_new(128);
```

Comment:
You'd think allocating the buffer just once (if and when it's needed) instead of on every `create_acl()` call would be more efficient. Empirically it either made no difference or performed slightly worse on `crm_simulate --profile` and `cts-cli -t acls` tests, compared to the current approach.

Reply:
It's possible glib is significantly less efficient than the standard functions, and/or it's sensitive to the initial size